assets/less/hel/fullwidth-hearing.less

```diff
@@ -0,0 +1,38 @@
+.fullwidth-hearing {
+  background-size: cover;
+  background-position: center center;
+  min-height: 250px;
+  padding: 30px;
+  width: 100%;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  &-header {
```

Comment:
Perhaps slight transparency for the grey header background would make the background image stand out a bit more? Alternatively, make the image element a bit taller than the header box. Don't know which option would look best.